src/components/TaskList/TaskList.module.scss

```diff
@@ -94,6 +94,7 @@
         align-items: center;
         flex-direction: column;
         color: #b6c2cf;
+        z-index: 120;
 
```

Comment:
do not write big z-indexes.
Write type of z-indexes in another file and export them